js/index.js

```diff
@@ -40,3 +40,70 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+// selecting nav items
+let nav = document.querySelectorAll('nav a');
+
+nav.forEach((item,i)=> item.innerText = siteContent.nav[`nav-item-${i+1}`]);
+
+const h1 = document.querySelector('h1');
+const h1Text = siteContent.cta['h1'];
+// inserting h1 text
+h1Text.split(' ')
+  .map(text => h1.innerHTML += `${text} <br>` )
+  .join('')
+
+  // inserting buttons
+  const btn = document.querySelector('.cta-text button');
+  btn.innerText = siteContent.cta['button'];
```

Comment:
This is fine, but it's preferable to use textContent instead of innerText when possible.
Check <a href="https://stackoverflow.com/questions/35213147/difference-between-textcontent-vs-innertext">this</a> and <a href="https://blog.cloudboost.io/why-textcontent-is-better-than-innerhtml-and-innertext-9f8073eb9061">this</a>